src/Employer/Employer.Web/Orchestrators/Part1/SearchResultPreviewOrchestrator.cs

```diff
@@ -38,7 +42,10 @@ public SearchResultPreviewOrchestrator(IVacancyClient client, IGeocodeImageServi
                 StartDate = vacancy.StartDate?.AsDisplayDate(),
                 LevelName = vacancy.Programme?.LevelName,
                 Title = vacancy.Title,
-                Wage = vacancy.Wage?.ToText()
+                Wage = vacancy.Wage?.ToText(
+                    () => _wageService.GetNationalMinimumWageRange(vacancy.StartDate.Value),
+                    () => _wageService.GetApprenticeNationalMinimumWage(vacancy.StartDate.Value)),
+                WageSuffix = (vacancy.Wage == null || vacancy.Wage.WageType == WageType.Unspecified) ? "" : "yearly"
```

Comment:
Could we have a `<span asp-condition="@Model.HasYearlyWage"> yearly</span>` in the view, and a `public bool HasYearlyWage` property in the view model instead of `WageSuffix`

Reply:
👍